main.css

```diff
@@ -0,0 +1,277 @@
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.new-line::after {
+  content: "\a";
+  white-space: pre;
+}
+
+img {
+  display: block;
+}
+
+header div img {
+  height: 16px;
+  width: 16px;
+}
+
+main {
+  padding-top: 70px;
+  display: block;
+}
+
+.flex {
+  display: flex;
+}
+
+.inline-flex {
+  display: inline-flex;
+}
+
+header {
+  height: 40px;
+  width: 100%;
+  justify-content: space-between;
+  position: fixed;
+  background-color: white;
+  z-index: 10;
+}
+
+#left-header {
+  width: 400px;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+#left-header button {
+  background-color: transparent;
+  border: none;
+  margin-left: 20px;
+  margin-right: 40px;
+}
+
+.main-image img {
+  width: 600px;
+}
+
+#mid-header img {
+  width: 200px;
+  height: 33px;
+  background-color: transparent;
+  padding-top: 3px;
+}
+
+#right-header button {
+  background-color: #567b95;
+  border: none;
+  margin-right: 15px;
+  padding: 0 10px;
+  height: 30px;
+  align-self: center;
+  border-radius: 3px;
+}
+
+#right-header {
+  width: 23%;
+}
+
+#title {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 55px;
+  font-style: italic;
+  font-family: georgia, 'times new roman', times, serif;
+  padding-bottom: 40px;
+  width: 70%;
+  text-align: center;
+  margin: 0 auto;
+}
+
+#section2-div2 button {
+  height: 35px;
+  align-self: center;
+  background-color: white;
+  color: black;
+  width: 120px;
+  border: black solid 1.5px;
+  border-radius: 2px;
+  font-size: 13px;
+  font-weight: bold;
+}
+
+#section2-div2 button:hover {
+  background-color: black;
+  color: white;
+}
+
+#section-1 h3 {
+  text-align: center;
+}
+
+#main-text {
+  font-family: georgia, 'times new roman', times, serif;
+  font-size: 22px;
+  line-height: 30px;
+}
+
+#second-div {
+  display: block;
+  width: 100%;
+}
+
+#main-image img {
+  width: 1000px;
+}
+
+#section-1 {
+  display: block;
+  width: 50%;
+  margin: 0 auto;
+}
+
+#section-2 {
+  display: block;
+  width: 30%;
+  margin: 0 auto;
+}
+
+#thank-youP {
+  text-align: center;
+}
+
+#inflation-img {
+  flex-direction: column;
+  margin: 0 0 0 -30%;
+}
+
+#inflation-img img {
+  height: 600px;
+  width: 900px;
+}
+
+#inflation-img h4 {
+  font-weight: bold;
+}
+
+#inflation-img p {
+  font-size: 13px;
+}
+
+iframe {
+  width: 100%;
+}
+
+figcaption {
+  display: inline;
+  font-size: 12px;
+}
+
+.credit {
+  font-size: 10px;
+}
+
+footer {
+  width: 100%;
+  padding-bottom: 20px;
+}
+
+#main-footer-div {
+  width: 50%;
+  margin: 0 auto;
+  justify-content: space-evenly;
+}
+
+footer ul {
+  list-style: none;
+}
+
+footer li:first-child {
+  font-weight: bold;
+  margin-bottom: 10px;
+}
+
+.top-subscribe {
+  font-weight: bold;
+  margin-bottom: 10px;
+}
```

Comment:
## Aim to remain DRY
- [ ] We appear to be repeating ourselves in lines `#193` & `#198`. Do you mind merging these rules?